Makefile

```diff
@@ -44,32 +62,54 @@ lint: ## Run golangci-lint
 verify: tidy generate format lint ## Verify the current code generation and lint
 	git diff --exit-code
 
-KUBERNETES_VERSION=v0.22.2
 COMBO_VERSION=$(shell git describe || echo $(DEFAULT_VERSION))
+BUILD_FLAGS=CGO_ENABLED=0
 VERSION_FLAGS=-ldflags "-X $(VERSION_PATH).GitCommit=$(GIT_COMMIT) -X $(VERSION_PATH).ComboVersion=$(COMBO_VERSION) -X $(VERSION_PATH).KubernetesVersion=$(KUBERNETES_VERSION)"
 build-cli: ## Build the CLI binary. Speciy VERSION_PATH, GIT_COMMIT, or KUBERNETES_VERSION to change the binary version.
-	$(Q)go build -a $(VERSION_FLAGS) -o ./bin/combo
+	$(Q) $(BUILD_FLAGS) go build $(VERSION_FLAGS) -o ./bin/combo
 
-IMAGE_REPO=quay.io/operator-framework/combo
-IMAGE_TAG=latest
 build-container: ## Build the Combo container. Accepts IMAGE_REPO and IMAGE_TAG overrides.
-	docker build . -f Dockerfile -t $(IMAGE_REPO):$(IMAGE_TAG)
+	docker build . -f Dockerfile -t $(IMAGE)
 
-CONTROLLER_GEN=$(Q)go run sigs.k8s.io/controller-tools/cmd/controller-gen
+build-local-container: build-cli ## Build the Combo container from the Dockerfile.local to speed compile time up. Accepts IMAGE_REPO and IMAGE_TAG overrides.
+	docker build . -f Dockerfile.local -t $(IMAGE)
 
-# Static tests.
-.PHONY: test test-unit verify build
+################
+# Test Targets #
+################
+.PHONY: test test-unit test-local test-e2e
 
-test: test-unit ## Run the tests
+test: test-unit test-e2e ## Run both the unit and e2e tests
 
+UNIT_TEST_DIRS=$(shell go list ./... | grep -v /test/)
 test-unit: ## Run the unit tests
-	$(Q)go test -count=1 -short ./...
+	$(Q)go test -count=1 -short $(UNIT_TEST_DIRS)
+
+test-e2e: ## Run the e2e tests
+	$(Q)go test -count=1 ./test/e2e
```

Comment:
(not blocking) We may need to migrate towards using ginkgo as the testing binary instead of `go test ...` at some point as there's some nuances around tests failing and the process existing before subsequent cleanup can be performed.